api/topology/src/main/java/quarks/topology/TStream.java

```diff
@@ -443,7 +443,7 @@ else if (WARNING.equals(lr.getLevel()))
     Set<String> getTags(); 
     
     /**
-     * Add an alias for the stream.
+     * Set an alias for the stream.
      * <p>
      * The alias must be unique within the topology.
```

Comment:
Is the statement "must be unique within the topology" too restrictive? 
The system registers a control associated with that TStream using ```ControlService.registerControl(String type, String id, String alias, Class<T> controlInterface, T control)```, where the 'alias' parameter is TStream's alias. The registration requires that 'alias' is unique within the context of the control 'type'.  So I could have:
```
TStream<Double> t1  = topology.poll(...);
t1.alias("foo");
// At run time, the PeriodicSource oplet registers a PeriodicMXBean with type "periodic" and alias "foo"
...
TStream<String> t2 = t1.map(...);
t2.alias("foo");
// At run time, the Map oplet registers a (hypothetical) MapMXBean with type "map" and alias "foo"
```